docs/src/main/paradox/patterns.md

```diff
@@ -3,5 +3,32 @@
 Many [Enterprise Integration Patterns](http://www.eaipatterns.com/toc.html) can be implemented with Akka Streams 
 (see [Java documentation](http://doc.akka.io/docs/akka/2.4/java/stream/index.html) or [Scala documentation](http://doc.akka.io/docs/akka/2.4/scala/stream/index.html)).
 
+
+# Splitter
+
+You can achieve a [Splitter](http://www.enterpriseintegrationpatterns.com/patterns/messaging/Sequencer.html) as described in  [EIP](http://www.enterpriseintegrationpatterns.com) using out of the box Akka Streams dsl.
+
+![Splitter](http://www.enterpriseintegrationpatterns.com/img/Sequencer.gif)
```

Comment:
We cannot hotlink images from other servers like this, we need to create our own image or ask permission to copy and host that image.

Reply:
I actually checked this and that Image is kindly provided under creative commons. (together with the pattern description). Camel links directly to it the docs and I did the same :). Your call guys here if Creative Commons is enough for you and if not we need to find another solution.

Reply:
I think that means we can include the image, but we don't want to link to an external http image because that will cause 'mixed content' errors in browsers when browsing the documentation over https (https://developer.lightbend.com/docs/alpakka/current/)

Reply:
In addition to using someone elses server bandwitdth. :)

Reply:
ok, I added the image locally